### PR TITLE
Update copyright and warranty commands to typescript

### LINKED ISSUE #680: Convert copyright and warranty commands to typescript
### Description
Convert copyright and warranty command to TypeScript

### Parent Issue
#649

### PR DESCRIPTION
**Pull request by shuse2**
### Description
Updating copyright and warranty commands to TypeScript.

### Review checklist

* The PR resolves #680 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

**Comment by shuse2** [comment]
esModuleInterop fixed by https://github.com/oclif/config/pull/61

File: src/commands/node/forging.ts [hunk 1]
```diff
@@ -30,7 +30,12 @@ interface Args {
 const STATUS_ENABLE = 'enable';
 const STATUS_DISABLE = 'disable';
 
-const processInput = (client: APIClient, status: string, publicKey: string, password?: string) => {
+const processInput = (
+	client: APIClient,
+	status: string,
+	publicKey: string,
+	password?: string,
+) => {
```
Comment (ishantiw) [inline src/commands/node/forging.ts]: missing return type

File: src/commands/transaction/create.ts [hunk 2]
```diff
@@ -53,7 +57,10 @@ const typeClassMap: TypeClassMap = {
 	multisignature: MultisignatureCommand,
 };
 
-const resolveFlags = (accumulated: ReadonlyArray<string>, [key, value]: [string, string | boolean | undefined]) => {
+const resolveFlags = (
+	accumulated: ReadonlyArray<string>,
+	[key, value]: [string, string | boolean | undefined],
+) => {
```
Comment (ishantiw) [inline src/commands/transaction/create.ts]: here as well

File: package.json [hunk 3]
```diff
@@ -139,13 +139,14 @@
...
 		"chai-as-promised": "7.1.1",
-		"coveralls": "3.0.0",
+		"coveralls": "3.0.2",
 		"globby": "8.0.1",
 		"husky": "0.14.3",
 		"lint-staged": "5.0.0",
 		"mocha": "4.0.1",
 		"nyc": "11.3.0",
 		"prettier": "1.9.2",
+		"rxjs-compat": "6.3.3",
```
Comment (mitsujutsu) [inline package.json]: is this being used?

Comment (shuse2) [reply package.json]: This is for `husky`, without this it seems like it's not working.
I have no idea why it doesn't work though...

**Pull request merged**